business_requirement_earned_value/report/br_earned_value_report.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import tools
+from openerp import fields, models
+
+
+class BusinessRequirementEarnedValueReport(models.Model):
+    _name = "business.requirement.earned.value.report"
+    _description = "Earned Value Report"
+    _auto = False
+
+    name = fields.Char('Name', readonly=True)
+    description = fields.Char('Description', readonly=True)
+    partner_id = fields.Many2one('res.partner', 'Customer',
+                                 readonly=True)
+    project_id = fields.Many2one('project.project', 'Master Project',
+                                 readonly=True)
+    res_product = fields.Many2one('product.product', 'Res Product',
+                                  readonly=True)
+    hr_timesheet_product =\
```

Comment:
@sudhir-serpentcs layout is not ok.
as example do as here:
hr_timesheet_product = fields.Many2one(
    'product.product',
    'HR Timesheet Product',
    ...,)

And amend for the others fields as well